tests/fixtures/project/docroot/sites/settings/testing.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+/**
+ * @file
+ * Only for testing purpose.
+ */
```

Comment:
Similarly this

Reply:
Fixture directory isn't added in the PR so one temp file is required.